playground/docker-compose.fork.yml

```
@@ -0,0 +1,172 @@
+version: "3.9"
+services:
+  chain:
+    image: ghcr.io/foundry-rs/foundry:latest
+    restart: always
+    entrypoint: /usr/local/bin/anvil
+    command: --fork-url ${ETH_RPC_URL} --block-time 12
```

I had to manually set a port that was **not** 8545 because `reth` was already listening on that port. Basically add this and replaced `8545` everywhere.
```suggestion
    command: --fork-url ${ETH_RPC_URL} --block-time 12 --port 8546
```

---

As above. Suggest to run this within docker-in-docker (ie. from within the devcontainer). This ensures that it is agnostic to services being run locally.